core/cat/plugins/quickstart_cat_plugin/models.py

```diff
@@ -72,20 +72,33 @@ def get_language_model(cat):
 
 @hook
 def get_language_embedder(cat):
-    # TODO: give more example ocnfigurations
+    # TODO: give more example configurations
 
     # Embedding LLM
-    using_openai_llm = type(cat.llm) in [OpenAI, OpenAIChat]
+    using_openai_llm = type(cat.llm) in [OpenAI, OpenAIChat, AzureOpenAI]
     if ("OPENAI_KEY" in os.environ) or using_openai_llm:
         openai_key = os.getenv("OPENAI_KEY")
         if openai_key is None:
             openai_key = cat.llm.openai_api_key
-        embedder = embedders.EmbedderOpenAIConfig.get_embedder_from_config(
-            {
-                "openai_api_key": openai_key,
-                # model_name: '....'  # TODO: allow optional kwargs
-            }
-        )
+        if using_openai_llm in [OpenAI, OpenAIChat]:
+            embedder = embedders.EmbedderOpenAIConfig.get_embedder_from_config(
+                {
+                   "openai_api_key": openai_key,
+                   # model_name: '....'  # TODO: allow optional kwargs
+                }
+            )
+        else:
+            print(cat.llm)
+            embedder = embedders.EmbedderAzureOpenAIConfig.get_embedder_from_config(
+                {
+                   "openai_api_key": openai_key,
+                   "model_name": "text-davinci-003",
+                   "api_base": "https://stregatto.openai.azure.com/",
```

Comment:
Can I read this value from `cat.llm.api_base` ? I was not able to figure out how to access the parameters